.cirrus.yml

```diff
@@ -126,6 +126,8 @@ build_task:
               VM_IMAGE_NAME: ${DEBIAN_CACHE_IMAGE_NAME}
               CI_DESIRED_RUNTIME: runc
               CI_DESIRED_NETWORK: netavark
+              # Ignore cgroups-v1 warnings on debian
+              IGNORE_CGROUPSV1_WARNING: true
```

Comment:
You probably also need to add this to https://github.com/containers/podman/blob/daf7a2c069a994f873d7c8b3409946318e674dcc/contrib/cirrus/lib.sh#L98